packages/web-client/app/components/card-pay/hub-authentication/index.hbs

```diff
@@ -12,7 +12,13 @@
           style={{css-var status-icon-size="2.5rem"}}
           data-test-failed
         >
-          Authentication failed or was canceled.
+          {{#if (eq this.error.message "AUTH_TIMEOUT")}}
+            <span>
```

Comment:
Needed to add `span` otherwise the layout gets broken with cardstack support link:
![image](https://user-images.githubusercontent.com/273660/139847979-758ec79f-22a7-4d6e-b5cf-de971674e8cb.png)